monitor/monitor.go

```diff
@@ -104,6 +94,36 @@ func RunMonitor(ctx context.Context, containerConfig build.ContainerConfig, relo
 					fmt.Fprint(stdout, helpMessage)
 				}
 			}
+			completer := func(d prompt.Document) []prompt.Suggest {
+				return []prompt.Suggest{}
+			}
+
+			prompt.New(exec, completer, prompt.OptionHistory([]string{
+				"reload", "rollback", "exit", "help",
+			}),
+				prompt.OptionPrefix("(buildx) "),
+				prompt.OptionParser(&customParser{
+					PosixParser: *prompt.NewStandardInputParser(),
+					reader:      in.stdin,
+				}),
+				prompt.OptionWriter(&posixWriter{
+					w: in.stdout,
+				}),
+				prompt.OptionAddASCIICodeBind(
+					prompt.ASCIICodeBind{
+						ASCIICode: []byte{0x1b, 'f'},
+						Fn: func(b *prompt.Buffer) {
+							b.CursorRight(1 + b.Document().FindEndOfCurrentWordWithSpace())
+						},
+					},
+					prompt.ASCIICodeBind{
+						ASCIICode: []byte{0x1b, 'b'},
+						Fn: func(b *prompt.Buffer) {
+							b.CursorLeft(len(b.Document().TextBeforeCursor()) - b.Document().FindStartOfPreviousWordWithSpace())
+						},
+					},
```

Comment:
What do these do? :eyes: I'm not familiar with these key combos. Are they particularly common?

Reply:
Move cursor with `word` increments